jolt-core/src/r1cs/spartan.rs

```diff
@@ -262,6 +267,25 @@ where
         })
     }
 
+    #[tracing::instrument(skip_all)]
```

Comment:
`prove_spartan_small_value` is already wrapped in `tracnig::instrument`. Why do we need this function at all?